Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -48,6 +48,9 @@ namespace Hazel {
 			s_GLFWInitialized = true;
 		}
 
+		#ifdef HZ_ENABLE_RENDERER_LOG
```

Comment:
The indent level here seems inconsistent with the other preprocessor conditions, was that intentional?

Reply:
Yes this was intentional.

Reply:
Yeah, it is not some conditional code, it is more like an if statement that has to hint glfw when we use opengl....